classes/UserDailyTasks.php

```diff
@@ -0,0 +1,125 @@
+<?php
+class UserDailyTasks {
+    const TASK_RESET = 86400;
```

Comment:
```suggestion
    const TASK_RESET_COOLDOWN = 86400;
```
Could be a little more specific with naming here